app/code/Magento/CatalogGraphQl/etc/module.xml

```diff
@@ -15,6 +15,7 @@
             <module name="Magento_GraphQl"/>
             <module name="Magento_StoreGraphQl"/>
             <module name="Magento_EavGraphQl"/>
+            <module name="Magento_Customer"/>
```

Comment:
Please, do not forget to add the corresponding dependency to the `composer.json`

Reply:
Hello @rogyar 
Thank you for your review.
This PR doesn't require to change the sequence. But, it requires adding module `magento/module-customer` into `require` section of the `composer.json` file.
I fixed it. Please recheck.
Thank you in advance.